src/_pytest/pytester.py

```diff
@@ -36,6 +36,8 @@
     u"/var/lib/sss/mc/passwd"
 ]
 
+CLOSE_STDIN = object
```

Comment:
Perhaps move this constant to the class for convenience? Then users can use `testdir.CLOSE_STDIN` instead of having to import from `_pytest.pytester`.